src/main/resources/org/jenkinsci/plugins/workflow/cps/replay/ReplayCause/description.jelly

```diff
@@ -28,10 +28,10 @@ THE SOFTWARE.
     <j:set var="original" value="${it.original}"/>
     <j:choose>
         <j:when test="${original != null}">
-            Replayed <a href="${rootURL}/${original.url}" class="model-link">#${it.originalNumber}</a> (<a href="${rootURL}/${it.run.url}replay/diff">diff</a>)
+            ${%Replayed_by_build(rootURL/original.url,it.originalNumber)} (<a href="${rootURL}/${it.run.url}replay/diff">${%diff}</a>)
```

Comment:
`rootURL/original.url` looks a little strange, do the expressions get coerced into strings and concatenated automatically? I would've expected `rootURL + "/" + original.url`. I'll double-check whether this works locally.

Reply:
I do another test again, and it's an issue. Already fixes.